src/actions/ast.js

```diff
@@ -92,7 +87,7 @@ export function setOutOfScopeLocations() {
 
     let locations = null;
     if (location.line && source && !source.isWasm && isPaused(getState())) {
-      locations = await findOutOfScopeLocations(
+      locations = await parser.findOutOfScopeLocations(
```

Comment:
adding some more specificity so that it is clear when we use a selector vs worker